cql3/statements/create_table_statement.cc

```diff
@@ -72,6 +72,30 @@ std::vector<column_definition> create_table_statement::get_columns() const
 
 future<std::tuple<::shared_ptr<cql_transport::event::schema_change>, std::vector<mutation>, cql3::cql_warnings_vec>>
 create_table_statement::prepare_schema_mutations(query_processor& qp, const query_options&, api::timestamp_type ts) const {
+    const auto tmptr = qp.proxy().get_token_metadata_ptr();
+    const auto& cfg = qp.db().get_config();
+    // FIXME: After adding a new rack, the keyspace is not RF-rack-valid anymore,
```

Comment:
Currently, the definition of the `--rf-rack-valid-keyspaces` option is like this:

```
    , rf_rack_valid_keyspaces(this, "rf_rack_valid_keyspaces", liveness::MustRestart, value_status::Used, false,
        "Enforce RF-rack-valid keyspaces. Additionally, if there are existing RF-rack-invalid "
        "keyspaces, attempting to start a node with this option ON will fail.")
```

...but the description will no longer be true after we merge your PR.

Please go over the uses of the RF-rack-valid term in the documentation and the user messages and make sure they make sense afterwards. Because "RF-rack-valid" is a term in the already released 2025.1 version, I think we should keep its meaning.

Reply:
We can also consider postponing it to a follow-up if it's not a quick fix but I'd rather have this fixed in this PR.

Reply:
I adjusted the uses for `RF-rack-valid` in the rebase to be consisted with the original meaning, but I believe we will want to actually change the meaning after we make it possible to fully handle keyspaces with RF < #racks by only using RF racks. Also, for that reason, I didn't create a new term for the state that `validate_rf_rack_valid_replication` is verifying